src/cms/templates/pages/page_form.html

```diff
@@ -472,4 +472,4 @@ <h2 class="heading font-normal">
     custom_file_field();
 }, false);
 </script>
-{% endblock %}
+{% endblock %}
```

Comment:
```suggestion
{% endblock %}

```
Do not remove new lines at the end of files